packages/flutter/lib/src/material/ink_well.dart

```diff
@@ -68,8 +68,8 @@ abstract class InteractiveInkFeature extends InkFeature {
   }
 }
 
-/// An encapsulation of an [InteractiveInkFeature] constructor used by [InkWell]
-/// [InkResponse] and [ThemeData].
+/// An encapsulation of an [InteractiveInkFeature] constructor used by
+/// [InkWell], [InkResponse], and [ThemeData].
```

Comment:
Why the ',' between ' [InkResponse]' and 'and'

Reply:
--> https://en.wikipedia.org/wiki/Serial_comma